hawkular-metrics/standalone.xml

```diff
@@ -277,8 +277,11 @@
               <replicated-cache name="schema" mode="SYNC">
                 <transaction mode="NON_XA"/>
               </replicated-cache>
-              <replicated-cache name="locks" mode="SYNC">
```

Comment:
The 'locks' cache was defined twice, correctly for hawkular-metrics container, and also by mistake in hawkular-alerts container.